performance_regression_test.py

```diff
@@ -155,6 +158,14 @@ def _clean_email_data():
         with open(email_data_path, 'w', encoding="utf-8"):
             pass
 
+    def _stop_load_after_one_nemesis_cycle(self):
+        time.sleep(300)  # wait 5 minutes to be sure nemesis has started
+        self.db_cluster.stop_nemesis(timeout=None)  # wait for Nemesis to end and don't start another cycle
```

Comment:
does `timeout=None` mean that it will wait forever?

Reply:
yes, if currently running nemesis does not end, this will be waiting forever.
But in that case `test_timeout` will break the process.

Reply:
yes, just wanted to be sure, that in this case, the timeout will be the test one, thanks